jasminewd2/index.d.ts

```diff
@@ -44,6 +44,13 @@ declare namespace jasmine {
     toThrowError(expected?: new (...args: any[]) => Error | Promise<new (...args: any[]) => Error>, message?: string | RegExp | Promise<string | RegExp>): Promise<void>;
   }
 
+  interface ArrayLikeMatchers<T extends ArrayLike<T>> extends Matchers<T> {
```

Comment:
If you have concern about using `ArrayLike` instead of `Array` I can investigate why this was initially used and replace it in the base jasmine typings as well